logstash-core/build.gradle

```diff
@@ -152,13 +152,16 @@ idea {
 def customJRubyDir = project.hasProperty("custom.jruby.path") ? project.property("custom.jruby.path") : ""
 def customJRubyVersion = customJRubyDir == "" ? "" : Files.readAllLines(Paths.get(customJRubyDir, "VERSION")).get(0).trim()
 
+String log4jVersion = '2.13.3'
```

Comment:
Do you think it worthwhile to move in top of the file and also define it in the `versions.yml`?

Reply:
not sure, what the benefit here would be. 
other (jar) dependencies are still versioned in *build.gradle* ... we might want to export all of them than?

Reply:
Ok, I agree on don't export the version in `versions.yml` since that stuff is somehow special, but I would move it on the top, to have all the versions properties one close to the other